src/test/codewhisperer/commands/basicCommands.test.ts

```diff
@@ -0,0 +1,121 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as assert from 'assert'
+import { beforeEach } from 'mocha'
+import * as sinon from 'sinon'
+import { resetCodeWhispererGlobalVariables } from '../testUtil'
+import { assertTelemetryCurried } from '../../testUtil'
+import { CodeWhispererConstants } from '../../../codewhisperer/models/constants'
+import { toggleCodeSuggestions, get, set } from '../../../codewhisperer/commands/basicCommands'
+import { FakeExtensionContext, FakeMemento } from '../../fakeExtensionContext'
+import { ExtContext } from '../../../shared/extensions'
+import { AwsContext } from '../../../shared/awsContext'
+import { SamCliContext } from '../../../shared/sam/cli/samCliContext'
+import { RegionProvider } from '../../../shared/regions/regionProvider'
+import { TelemetryService } from '../../../shared/telemetry/telemetryService'
+import { CredentialsStore } from '../../../credentials/credentialsStore'
+import { testCommand } from '../../shared/vscode/testUtils'
+import { Command } from '../../../shared/vscode/commands2'
+
+describe('CodeWhisperer-basicCommands', function () {
+    describe('CodeWhisperer-basicCommands', function () {
```

Comment:
Duplicated?

Reply:
no lol, I saw there are some other tests with this format, so...

Reply:
let me remove it

Reply:
> no lol, I saw there are some other tests with this format, so...

Those probably slipped through code review :)